src/main/java/org/gitlab4j/api/models/Discussion.java

```diff
@@ -0,0 +1,48 @@
+package org.gitlab4j.api.models;
+
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import org.gitlab4j.api.utils.JacksonJson;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class Discussion {
+
+  private String id;
+  private String individualNote;
```

Comment:
This should be a Boolean, not a String.  Will need to update the set and get methods also.